index.html

```diff
@@ -0,0 +1,285 @@
+      <h4>Frameworks</h4>
+      <ul class="menu-frameworks">
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+        <li>RSPec</li>
+        <li>CapyBara</li>
+        <li>Selenium</li>
+      </ul>
+    </div>
+
+    <div class="Skills">
+      <h4>Skills</h4>
+      <ul class="menu-skills">
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </div>
+  </div>
+
+      
+    
+
+<!-- line2 -->
+  <div class="hr" style="padding:20px; margin: 20px; color: dark">
+    <hr>
+  </div>
+            <!-- social media -->
+            <div class="section-one-column2">
+              <ul class="socialmedia">
+                  <li><a href=""><img src="./img/Vector.png" alt="github"></a></li>
+                  <li><a href=""><img src="./img/Frame.png"alt="Linkedin"></a></li>
+                  <li><a href=""><img src="./img/Vector5.png" alt="angel"></a></li>
+                  <li><a href=""><img src="./img/twitter.png" alt="twitter"></a></li>
+                  <li><a href=""><img src="./imG/M.png"alt="medium"></a></li>
+              </ul>
+            </div>
+
+            <!-- flesh down -->
+            <div class="flesh-down">
+              <img src="./img/Group 55.png" alt="flesh-down">
+              <img src="./img/Work.png" alt="work">
+            </div>
+
+            <!-- My recent work -->
+            <div class="My-Recent-Work">
+              <h3>MY RECENT WORK</h3>
+            </div>
+          
+            <!-- line3 -->
+            <div class="hr" style="padding:20px; margin: 20px; color: dark">
+              <hr>
+            </div>
+
+            <!--PLACE HOLDER IMAGE-->
+            <div class="place-holder-img">
+              <img src="./img/ImgPlaceholder.png" alt="place holder">
+            </div>
+
+            <!-- MUTLI POST STORIES -->
+            <div class="title-post">
+              <h2>Multi-Post Stories</h2>
+            </div>
+            <div class="paragraph2">
+              <p>A daily selection of privately <br>
+                 personalized reads; no accounts <br>
+                 or sign-ups required. has been the <br>
+                 industry's standard dummy text <br>
+                 ever since the 1500s, when an <br>
+                 unknown printer took a standard <br>
+                 dummy text.</p>
+            </div>
+
+            <!-- languajes de programacion -->
+            <ul class="programming-languages">
+              <li><a href="">CSS</a></li>
+              <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+              <li><a href="">html</a></li>
+              <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+              <li><a href="">bootstrap</a></li>
+              <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+              <li><a href="">Ruby</a></li> 
+            </ul>
```

Comment:
- [ ] [OPTIONAL] The `<a>` tag is not necessary for the languages list since these are not links